src/mirage/index.ts

```diff
@@ -69,6 +69,25 @@ export function makeServer(baseUrl: string) {
       this.get('https://mantis.us-east-1.test.netflix.net/library/list', (schema) => {
         return schema.db.artifacts;
       });
+
+      this.delete('/v1/jobs/:jobId', (schema, request) => {
+        const { jobId } = request.params; // jobId to be deleted
+
+        schema.db.jobs.forEach((job: Job) => {
```

Comment:
Also this API should not remove the job but simply mark it as deleted by updating the state.

Reply:
Corrected.